xbanish: add page (#22688)
tldr-pages/tldr: pages/linux/xbanish.md

# xbanish

> > X utility that hides the cursor when a key is pressed, the cursor appears again when there is clicking or movement.

Reviewer: https://github.com/tldr-pages/tldr/blob/8dbfde07690ba3ec2d93cee74c16b1b1eba99e00/contributing-guides/style-guide.md#imperative-mood
> All descriptions must be phrased in the imperative mood.

```suggestion
> Hide the X11 cursor when a key is pressed and turn it back on when the mouse is used.
```

> > More information: <http://manned.org/xbanish>.

Reviewer: ```suggestion
> More information: <https://manned.org/xbanish>.
```

- Hide the cursor on key press:

`xbanish`

> - Start xbanish so that the cursor is always hidden, ignoring key presses:

Reviewer: ```suggestion
- Always keep the cursor hidden, ignoring key presses and mouse usage:
```

`xbanish -a`

- Automatically hide the cursor after a certain amount of time of inactivity:

> `xbanish -t {{seconds}}`

Reviewer: ```suggestion
- Automatically hide the cursor after `n` seconds of inactivity:

`xbanish -t {{n}}`
```
